src/docfx/Models/BuildCommandOptions.cs

```diff
@@ -60,6 +60,9 @@ internal class BuildCommandOptions : ICanPrintHelpMessage, ILoggable
         [Option('f', "force", HelpText = "Force re-build all the documentation")]
         public bool ForceRebuild { get; set; }
 
+        [Option("forcePostProcess", HelpText = "Force to re-process the documentation in post processors. It will be cascaded from force option.")]
+        public bool ForcePostProcess { get; set; }
```

Comment:
I think 'process' here is a verb? like 'force to post process', how do you think?

---
In reply to: [94204150](https://github.com/dotnet/docfx/pull/1091#discussion_r94204150) [](ancestors = 94204150)